purchase_order_line_schedule_manual_delivery/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,6 @@
+The goal of this module is to allow the manual creation of incoming shipments.  When installed,
+a purchase order won't impact directly the stock. It will not create the pickings upon order confirmation.
+It allows the reception and the impact on stock to be done manually when needed.
+The goal is to be used when a supplier does not confirm the full order at once. As the supplier
+confirms the different purchase order lines, the user can manually create the incoming
+shipments for those specific lines.
```

Comment:
The goal of this module is to allow the manual creation of incoming shipments.  When installed,
a purchase order won't impact the stock directly. It will not create stock pickings upon order confirmation.
It allows the reception and the impact on stock to be done manually when needed.
The goal is to be used when a supplier does not confirm the full order at once. As the supplier
confirms different purchase order lines, the user can manually create the incoming
shipments for those specific lines.